packages/lib/src/vis/shared/Axis.tsx

```diff
@@ -45,11 +45,18 @@ function Axis(props: Props) {
   const isAbscissa = type === 'abscissa';
   const axisLength = isAbscissa ? width : height;
 
-  const { scaleType = ScaleType.Linear, isIndexAxis, showGrid, label } = config;
+  const {
+    scaleType = ScaleType.Linear,
+    isIndexAxis,
+    showGrid,
+    label,
+    nice: _nice = false,
+  } = config;
   // Restrain ticks scales to visible domains
   const scale = createAxisScale(scaleType, {
     domain,
     range: flipAxis ? [axisLength, 0] : [0, axisLength],
+    nice: _nice,
```

Comment:
Any particular reason for renaming the variable here and below?

Reply:
Not really. I will tidy up and use literal property value shorthands too.